src/archi/Aggregate.ts

```diff
@@ -0,0 +1,7 @@
+import { DomainEvent } from './DomainEvent';
+
+export type PublishEvent = (event: DomainEvent) => void;
+
```

Comment:
Ajout d'un type pour simplifier l'écriture `string | string[]` pour aggregateId : 
```suggestion

export type AggregateId = string | string[];

```

Reply:
Oui, ce sera sans doute utile d’avoir un type pour l’aggregateId (pour les autres méthodes qui les manipulent). 
En revanche,